src/lib/QueryRenderer.ts

```diff
@@ -0,0 +1,53 @@
+import type { IQuery } from '../IQuery';
+import { getGlobalQuerySource } from '../Config/Settings';
+import { GlobalFilter } from '../Config/GlobalFilter';
+import { Query } from '../Query/Query';
+
+/**
+ * @summary
+ * This file contains utilities used by {@link QueryRenderer} that should actually
+ * be in that file. But the file {@link QueryRenderer} is in depends on dependencies
+ * that aren't available during automated testing, and would make it impossible to run
+ * those tests on the code that is instead housed here.
+ */
+
+/**
+ * Explains a query rendered by {@link QueryRenderer}
+ *
+ * Specifically, returns a formatted string:
+ *     * Explains whether a global filter is in use
+ *     * Explains whether the global query if it's in use
+ *     * Explains the query described by {@link source}
+ *
+ * @param {string} source The source of the task block to explain
+ * @returns {string}
+ */
+export function explainResults(source: string): string {
+    let result = '';
+
```

Comment:
> I disagree (but not strongly enough to avoid making any changes you'd prefer to see here).
> 
> * `Explanation.ts` seems to be concerned with: "How do I represent the explanation of a Query Instruction"
> * whereas `explainResults` is responsible for generating the explanation of the results of a task block, which may involve upto 2 queries (global + block) and the global filter.
> 
> The differences in scope of responsibilities made it seem to me that they should be seperate.

I take your point, and also don't feel super strongly about it. Am happy with whatever you decide to do...